third_party/amd/backend/compiler.py

```diff
@@ -52,7 +52,7 @@ def __post_init__(self):
         # Ignore user-defined warp size for gfx9
         warp_size = 32 if 'gfx10' in self.arch or 'gfx11' in self.arch else 64
         object.__setattr__(self, 'warp_size', warp_size)
-        libs = ["cuda2gcn", "opencl", "ocml", "ockl"]
+        libs = ["ocml", "ockl"]
```

Comment:
@antiagainst Did you leave opencl for some reason when you cleaned up the bc files?

Reply:
No particular reason. I was just dropping the ones obviously not needed before due to my change. Didn't inspect whether this one is still needed. Great to know that it can be dropped too!